tests/management/conftest.py

```diff
@@ -5,23 +5,14 @@
 from io import BytesIO
 from unittest.mock import patch
 
-import boto3
 import botocore.client
-import moto
 import pytest
 
 from cirrus.lib.process_payload import ProcessPayload, ProcessPayloads
 from cirrus.management.cli import cli
 from cirrus.management.deployment_pointer import DEPLOYMENTS_PREFIX
 from click.testing import CliRunner
 
-
-@pytest.fixture()
-def timestream_write_client():
-    with moto.mock_aws():
-        yield boto3.client("timestream-write", region_name="us-east-1")
```

Comment:
We're not using this to mock timestream for testing??? I mean, apparently not? You can close this if the answer is yes, I just felt like this change is not obvious so I wanted to confirm.